keyboards/melody96/melody96.h

```diff
@@ -0,0 +1,28 @@
+#ifndef KB_H
```

Comment:
(Optional) Could you change this to `MELODY_H` or the like, so it's not generic?